packages/react/src/SectionIntro/SectionIntro.tsx

```diff
@@ -0,0 +1,79 @@
+import React, {forwardRef, type Ref} from 'react'
+import clsx from 'clsx'
+import {Button, type ButtonProps} from '../Button'
+import {Heading, HeadingTags} from '../Heading'
+import {Text} from '../Text'
+import styles from './SectionIntro.module.css'
+
+import type {BaseProps} from '../component-helpers'
+
+export type SectionIntroProps = BaseProps<HTMLDivElement> & {
+  children: React.ReactNode | React.ReactNode[]
```

Comment:
You should be able to avoid redeclaring `children` each time by doing this:

```jsx
export type SectionIntroProps = BaseProps<HTMLDivElement> & PropsWithChildren & {
```